Composer/packages/client/src/components/WebChat/WebChatPanel.tsx

```diff
@@ -50,35 +54,50 @@ export const WebChatPanel: React.FC<WebChatPanelProps> = ({
   const conversationService = useMemo(() => new ConversationService(directlineHostUrl), [directlineHostUrl]);
   const webChatPanelRef = useRef<HTMLDivElement>(null);
   const [currentRestartOption, onSetRestartOption] = useState<RestartOption>(RestartOption.NewUserID);
-  const directLineErrorChannel = useRef<WebSocket>();
+  const webChatTrafficChannel = useRef<WebSocket>();
 
   useEffect(() => {
     const bootstrapChat = async () => {
       const conversationServerPort = await conversationService.setUpConversationServer();
       try {
-        directLineErrorChannel.current = new WebSocket(
-          `ws://localhost:${conversationServerPort}/ws/errors/createErrorChannel`
-        );
-        if (directLineErrorChannel.current) {
-          directLineErrorChannel.current.onmessage = (event) => {
-            const data: DirectLineLog = JSON.parse(event.data);
-            appendLogToWebChatInspector(projectId, data);
-            setTimeout(() => {
-              setActiveTabInDebugPanel('WebChatInspector');
-              setDebugPanelExpansion(true);
-            }, 300);
+        // set up Web Chat traffic listener
+        webChatTrafficChannel.current = new WebSocket(`ws://localhost:${conversationServerPort}/ws/traffic`);
+        if (webChatTrafficChannel.current) {
+          webChatTrafficChannel.current.onmessage = (event) => {
+            const data:
+              | ConversationActivityTraffic
+              | ConversationNetworkTrafficItem
+              | ConversationNetworkErrorItem = JSON.parse(event.data);
+            if (data.trafficType === 'network') {
+              appendWebChatTraffic(projectId, data);
+            }
+            if (data.trafficType === 'activity') {
+              appendWebChatTraffic(
+                projectId,
+                data.activities.map((a) => ({ activity: a, timestamp: a.timestamp, trafficType: data.trafficType }))
+              );
+            }
+            if (data.trafficType === 'networkError') {
+              appendWebChatTraffic(projectId, data);
+              setTimeout(() => {
```

Comment:
why the timeout?

Reply:
this was the behavior of the component before I added to it, so I decided to leave it as is